frontend/macos/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -15,7 +15,7 @@
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "33CC10EC2044A3C60003C045"
```

Comment:
Not sure what this is, if this is a key this should be in `.env` file instead.

Reply:
Added sensitive keys: commit:  2a6e78675aafe70458f5179aa564457e05e65eec
specifically macOS BuildableReference Identifiers in .env file

1. created .env file in frontend folder
2. added *.env in both .gitignore
3. import package flutter_dotenv int main.dart
4. added DotEnv variable  in main.dart to load the key
5. flutter pub get to get flutter_dotenv dependency (flutter_dotenv^5.0.0)

result: app still functions